### ISSUE TITLE
Should have aggregates assertion

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/activefedora-aggregation#2]
```
+   <http://www.openarchives.org/ore/terms/aggregates> 
+           <http://127.0.0.1:8983/fedora/rest/dev/file2>,
+           <http://127.0.0.1:8983/fedora/rest/dev/file1> ;
```

**Comment by elrayle** [comment]
For PCDM, should have <PCDM:hasMembers> instead of <ORE:aggregates>.

**Comment by jcoyne** [comment]
Fixed by 7ffbaccfbda9bfd902e676cf3d8b66f5f7ff8332
